spec/views/idv/cancellations/destroy.html.erb_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+describe 'idv/cancellations/destroy.html.erb' do
+  let(:hybrid_session) { false }
+  let(:sp_name) { nil }
+  let(:return_to_sp_path) { root_url }
+
+  before do
+    decorated_session = instance_double(ServiceProviderSessionDecorator)
+    allow(decorated_session).to receive(:sp_name).and_return(sp_name)
```

Comment:
we can include this in the declaration

```suggestion
    decorated_session = instance_double(ServiceProviderSessionDecorator, sp_name: sp_name)
```

Reply:
Good call, thanks! 👍